alf/utils/value_ops.py

```diff
@@ -118,6 +118,72 @@ def action_importance_ratio(action_distribution,
     return importance_ratio, importance_ratio_clipped
 
 
+def generalized_advantage_estimation(rewards,
```

Comment:
I didn't change ``generalized_advantage_estimation``, only moved it closer to action_importance_ratio, git messed up the diff..